### ISSUE TITLE
Fill out details under author section 

### ISSUE THREAD

**Issue by advaithm** [lionloya/NightPDF#11]
I used placeholders because these sections were not filled. For deb files the author name and email are required. Please fill them out before making a release

**Comment by lionloya** [comment]
Yeah man, I'll try to build it tomorrow and test it on Linux. Thanks for the heads up

**Comment by advaithm** [comment]
to build it on a linux system you will need dpkg and rpm installed
